hack/update-attributions.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+set -eoux pipefail
+
+SPLIT="**********************************************************************"
```

Comment:
Either

```suggestion
SPLIT=---"
```
or

```suggestion
SPLIT=***"
```

would give the same result while having less characters: https://www.markdownguide.org/basic-syntax/#horizontal-rules

Reply:
Replaced it with the latter.